### ISSUE TITLE
librispeech deepspeech torch workload NCCL timeout bug

### ISSUE THREAD

**Issue by sourabh2k15** [mlcommons/algorithmic-efficiency#245]
librispeech deepspeech torch workload eval hits a NCCL error 
rerunning now to see if error occurs again 

error:

```
 File "/home/smedapati/.local/lib/python3.8/site-packages/torch/distributed/distributed_c10d.py", line 2136, in _all_gather_bas[269/1974]
    work = group._allgather_base(output_tensor, input_tensor)                                                                            
RuntimeError: NCCL communicator was aborted on rank 0.  Original reason for failure was: [Rank 0] Watchdog caught collective operation tim
eout: WorkNCCL(SeqNum=472268, OpType=_ALLGATHER_BASE, Timeout(ms)=3600000) ran for 3608061 milliseconds before timing out.                
E1103 23:25:09.147905 140390057297728 submission_runner.py:369] Eval step 20510 error.                                                    
Traceback (most recent call last):                                                                                                        
  File "submission_runner.py", line 337, in train_once                                                                                    
    latest_eval_result = workload.eval_model(global_eval_batch_size,                                                                      
  File "/home/smedapati/algorithmic-efficiency/algorithmic_efficiency/spec.py", line 307, in eval_model                                  
    train_metrics = self._eval_model_on_split(                                                                                            
  File "/home/smedapati/algorithmic-efficiency/algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py", li
ne 286, in _eval_model_on_split                                                                                                          
    dist.all_reduce(metric)                                                                                                              
  File "/home/smedapati/.local/lib/python3.8/site-packages/torch/distributed/distributed_c10d.py", line 1320, in all_reduce              
    work = default_pg.allreduce([tensor], opts)                                                                                          
[work2] 0:[tmux]*                                      
    work = default_pg.allreduce([tensor], opts)                                                                                 [255/1974]
RuntimeError: NCCL communicator was aborted on rank 5.  Original reason for failure was: [Rank 5] Watchdog caught collective operation tim
eout: WorkNCCL(SeqNum=472268, OpType=ALLREDUCE, Timeout(ms)=3600000) ran for 3607640 milliseconds before timing out.
Traceback (most recent call last):
  File "submission_runner.py", line 574, in <module>
    app.run(main)
  File "/home/smedapati/.local/lib/python3.8/site-packages/absl/app.py", line 312, in run
    _run_main(main, args)
  File "/home/smedapati/.local/lib/python3.8/site-packages/absl/app.py", line 258, in _run_main
    sys.exit(main(argv))
  File "submission_runner.py", line 547, in main
    score = score_submission_on_workload(workload,
  File "submission_runner.py", line 485, in score_submission_on_workload
    timing, metrics = train_once(workload, global_batch_size,
  File "submission_runner.py", line 307, in train_once
```

**Comment by runame** [comment]
On which branch are you running this?

**Comment by pomonam** [comment]
This should be fixed by #251 .
